src/module-1/cards-list-v1/stylesheets/card.css

```diff
@@ -0,0 +1,113 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100;300&display=swap');
+*{
```

Comment:
consider to follow consistent formatting:  
```
.class1 {  
  attr: 1;
  attr: 2;
}

.class2 {  
  attr: 1;
  attr: 2;
}

Reply:
[Modified](https://github.com/front-camp-2021/html-css-module-Kyanka/commit/a96e851a8ea308063ab34f162ef68b4714e0db1e)